ios/FluentUI.Demo/FluentUI.Demo/Demos/PopupMenuDemoController_ObjC.h

```diff
@@ -0,0 +1,15 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface PopupMenuDemoController_ObjC : UIViewController
```

Comment:
hmmm feels weird we need to have _objc appended.

Reply:
I mostly did that to follow all the _SwiftUI examples we had.